src/test/kotlin/build/buf/gradle/ConfigurationTest.kt

```diff
@@ -44,7 +44,6 @@ class ConfigurationTest : AbstractBufIntegrationTest() {
     }
 
     private fun assertSuccess() {
-        val result = gradleRunner().withArguments(":tasks").build()
-        assertThat(result.output).doesNotContain("cannot use both the protobuf-gradle-plugin and a Buf workspace")
+        assertThat(gradleRunner().withArguments(":tasks").build().output.contains("BUILD SUCCESSFUL"))
```

Comment:
I think `build()` itself verifies success, but this doesn't hurt?

Reply:
I guess there's no need for any kind of assert in that case?

Reply:
Fixed

Reply:
Did you happen to verify that no assertion was needed (like tweaking the config files to expect failure and making sure the test fails)? If so, LGTM!

Reply:
Yes, I was able to verify that the test will fail if there is any sort of build error in the resources dir. I also am adding a couple more test cases to nail things down.